CAFMaker/CAFMaker.fcl

```diff
@@ -9,7 +9,7 @@ standard_cafmaker:{
     SSDRawLabel:  "raw:SSD"
     SSDClustLabel: "ssdclusts"
     BACkovHitLabel: "backovreco"
-    GetMCTruth:    true
+    GetMCTruth:    false
```

Comment:
This changes the expected behaviour for anyone working with MC. Might be worth a message in #emphatic_software or switching it back. I do have looking into an actual fix on my todo list, but I don't know when I'll get to it.

Reply:
added comments to both GetMCTruth options describing when to use which